doc/folders.md

```diff
@@ -425,49 +426,50 @@ Collection<BoxCollaboration.Info> collaborations = folder.getCollaborations();
 
 [get-collaborations]: https://box.github.io/box-java-sdk/javadoc/com/box/sdk/BoxFolder.html#getCollaborations--
 
-Set Metadata
-------------
+Create Metadata
+---------------
 
-To set metadata on a folder, call [`setMetadata(String templateKey, String templateScope, Metadata properties)`][set-metadata].
+Metadata can be created on a folder by calling
+[`createMetadata(Metadata metadata)`][create-metadata],
+[`createMetadata(String templateName, Metadata metadata)`][create-metadata-2], or
+[`createMetadata(String templateName, String scope, Metadata metadata)`][create-metadata-3]
 
+<!-- sample post_folders_id_metadata_id_id -->
 ```java
 BoxFolder folder = new BoxFolder(api, "id");
-folder.setMetadata("test_template", "enterprise", new Metadata().add("/foo", "bar"));
+folder.createMetadata(new Metadata().add("/foo", "bar"));
 ```
 
-Note: This method will unconditionally apply the provided metadata, overwriting existing metadata for the keys provided.
-To specifically create or update metadata, please refer to the `createMetadata()` and `updateMetadata()` methods.
+Note: This method will only succeed if the provided metadata template is not currently applied to the folder, otherwise
+it will fail with a Conflict error. To get to know how to edit existing metadata please
+go to [set metadata](#set-metadata) and [update metadata](#update-metadata) sections.
 
-Metadata can be created on a folder by calling
-[`createMetadata(Metadata properties)`][create-metadata],
-[`createMetadata(String templateKey, Metadata properties)`][create-metadata-2], or
-[`createMetadata(String templateKey, String templateScope, Metadata properties)`][create-metadata-3]
+[create-metadata]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFolder.html#createMetadata-com.box.sdk.Metadata-
+[create-metadata-2]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFolder.html#createMetadata-java.lang.String-com.box.sdk.Metadata-
+[create-metadata-3]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxFolder.html#createMetadata-java.lang.String-java.lang.String-com.box.sdk.Metadata-
 
-Note: This method will only succeed if the provided metadata template is not currently applied to the folder, otherwise
-it will fail with a Conflict error.
+Set Metadata
+------------
+
+To set metadata on a folder, call [`setMetadata(String templateName, String scope, Metadata metadata)`][set-metadata].
```

Comment:
In files we have this part added:
```
This method will try to create provided metadata on a file. However, if metadata has already been applied 
to this file, it will overwrite values of metadata keys specified in the `metadata` parameter.
```
Maybe we could copy it here as well?
Should we say that it will execute 2 API calls?
Should we say even clearer that if you do not change metadata fields they will remain unchanged? It says it will change values specified but some users were confused.